index.js

```diff
@@ -270,7 +263,7 @@ Parser.prototype = {
               type: 'Code',
               val: tok.val,
               buffer: tok.buffer,
-              escape: tok.escape,
+              mustEscape: tok.mustEscape !== undefined ? tok.mustEscape : tok.escape,
```

Comment:
I'd prefer just `tok.mustEscape !== false` here.  That fails to safe.  Alternatively (perhaps as well) we could add an `assert(typeof tok.mustEscape === 'boolean')` check before this.  I want to ensure everyone is on a compatible version, and that if they aren't then it's not dangerous, rather than trying to necessarily just work.